File: plugin/src/org/jetbrains/yesod/julius/JuliusLanguage.java
package org.jetbrains.yesod.julius;

/**
 * @author Leyla H
 */

import com.intellij.lang.Language;

public class JuliusLanguage extends Language {
    public static final JuliusLanguage INSTANCE = new JuliusLanguage();

    public JuliusLanguage() {
        super("Julius", "text/julius");
    }
}

File: plugin/src/org/jetbrains/yesod/julius/JuliusLanguage.kt
package org.jetbrains.yesod.julius

/**
 * @author Leyla H
 */

import com.intellij.lang.Language

class JuliusLanguage : Language("Julius", "text/julius") {
    companion object {
        val INSTANCE: JuliusLanguage = JuliusLanguage()
    }
}